tests/at/main.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2018 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    AT module test application
+ *
+ * @author   Vincent Dupont <vincent@otakeys.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "at.h"
+#include "shell.h"
+#include "timex.h"
+
+static at_dev_t at_dev;
+static char buf[256];
+static char resp[1024];
+
+#ifndef UART_AT
+#define UART_AT     UART_DEV(1)
+#endif
+
+#ifndef BAUDRATE_AT
+#define BAUDRATE_AT 115200
+#endif
+
+static int init(int argc, char **argv)
+{
+    (void)argc;
+    (void)argv;
+
+    at_dev_init(&at_dev, UART_AT, BAUDRATE_AT, buf, sizeof(buf));
```

Comment:
I think, it would be more flexible to pass the UART and baudrate to the init function.